sherpa/ui/utils.py

```diff
@@ -4069,6 +4069,8 @@ def delete_data(self, id: Optional[IdType] = None) -> None:
         -----
         The source expression is not removed by this function.
 
+        The routine does nothing if the given dataset does not exist.
+
```

Comment:
I know it's just a doc change, so this is not a request to change anything, but I general I think it's not good practice to fail silently.